src/zadatak9/Nekretnina.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * 
+ */
+package zadatak9;
+
+/**
+ * @author Radovan.Olujic
+ *
+ */
+public abstract class Nekretnina {
+
+	private String adresa;
+	private int zona;
+	private double kvadratura;
+	private Vlasnik vlasnik;
+
+	protected Nekretnina() {
+		super();
```

Comment:
da li ti ovde treba "super();"?

Reply:
Ne, ali to sam Eclypse ubacuje, ka generisem konstruktor...